### PR TITLE
Docker compose web multi sites on Apach2/nginx

### PR DESCRIPTION
**Pull request by ec49ce**
to use multi site feaches 

on apache2 need:
create config file for domain
put it in to /etc/apache2/sites-available/ 
exec a2ensite \<config file for domain\>

on nginx need:
create config file for domain
put it in to /etc/nginx/conf.d/

### CODE REVIEW

File: Docker-compose-web-srv/debian-apache2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.10
+
+RUN apt-get update -y; \
+    apt-get install -y apt-utils; \
```
Comment (drempost) [inline Docker-compose-web-srv/debian-apache2/Dockerfile:4]: use one apt-get install instead of three:
apt-get install -y apt-utils apache2 apache2-utils

**Review by drempost — APPROVED** [review]
fine. but could be a bit improved

File: Docker-compose-web-srv/debian-apache2/Dockerfile (hunk 1 above)
Comment (ec49ce) [reply Docker-compose-web-srv/debian-apache2/Dockerfile:4]: I completely agree. At some moment I kept it in my mind, but forgot at the last stage - when everything worked.

**Review by drempost — APPROVED** [review]
now it looks better, you are welcome to merge

**Pull request merged**